ui/src/main/webapp/src/app/components/project-list/project-list.component.ts

```diff
@@ -56,12 +54,6 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     ngOnInit():any {
         this.updateSort();
         this.getMigrationProjects();
-        this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
```

Comment:
Ok, next time I will make sure to write about such change explicitely to the PR description or highlight by a comment.